TurkishFilter/sections/specific.txt

```diff
@@ -2203,6 +2203,7 @@ itemci.com##.header_rek
 itemci.com##a[href="https://www.astekmt2.net/"]
 hdplayer.org###adversing
 hdplayer.org###player > .alert_msg
+frmtr.com##img[src^="https://www.frmtr.com/images/x"]
```

Comment:
please remove this rule too, regexp is enough here

Reply:
8249dbc55a2da08cbefa320d5304d99e2142ea84